ocs_ci/utility/utils.py

```diff
@@ -1454,3 +1458,25 @@ def create_rhelpod(namespace, pod_name):
     )
     helpers.wait_for_resource_state(rhelpod_obj, constants.STATUS_RUNNING)
     return rhelpod_obj
+
+
+def timeout_reached(start_time, timeout, err_msg=None):
+    """
+    Check if timeout reached and if so raise the exception.
+
+    Args:
+        start_time (time): Star time of the operation.
+        timeout (int): Timeout in seconds.
+        err_msg (str): Error message for the exception.
+
+    Raises:
+        TimeoutException: In case the timeout reached.
+
+    """
+
```

Comment:
remove blank line

Reply:
After end of each section there is suppose to be one extra free line. See also here for examples: http://www.sphinx-doc.org/en/1.4/ext/example_google.html you will see one free line. People always forgot to add it as I see in our code but we should follow this!

Reply:
Ah, after offline discussion I realized that you do not mean the this line for 1474 but after the docstring. Then yep, this space is not necessary but also nothing bad. But I can remove this line ;) thanks